howso/utilities/feature_attributes/infer_feature_attributes.py

```diff
@@ -301,11 +301,16 @@ def infer_feature_attributes(data: Union[pd.DataFrame, SQLRelationalDatastorePro
                 "size" : [ "small", "medium", "large", "huge" ]
             }
 
-    dependent_features: dict, default None
-        (Optional) Dict of features with their respective lists of features
-        that either the feature depends on or are dependent on them. Should
-        be used when there are multi-type value features that tightly
-        depend on values based on other multi-type value features.
+    dependent_features : dict, optional
+        Dict of feature names to a list of feature(s) that they depend on or
```

Comment:
I find the compound plurals confusing. This may help:

> Dict mapping a feature to a list of other feature(s) that it depends on...

I also think there should be some mention of the fact that, especially with continuous features, there is still a probabilistic chance a value will be generated that is correlated to the dependent feature in the original data.